rasa/shared/core/domain.py

```diff
@@ -1868,6 +1868,7 @@ def is_domain_file(filename: Text) -> bool:
         try:
             content = rasa.shared.utils.io.read_yaml_file(filename)
         except (RasaException, YamlSyntaxException):
+            logger.warning(f"The file {filename} could not be loaded as domain file. You can use https://yamlchecker.com/ to validate the YAML syntax of your file.")
```

Comment:
We actually follow [Python's recommendation](https://docs.python.org/3.8/howto/logging.html#when-to-use-logging) to use `warnings.warn` instead of `logger.warning` because an invalid domain would be an usage error. 
There's a very useful utility function for this: [`rasa.shared.utils.io.raise_warning`](https://github.com/RasaHQ/rasa/blob/761cfc3376383c40fe93879a5e79646a0ffa10e3/rasa/shared/utils/io.py#L51).

Reply:
Also please resolve the `E501 line too long` warning in this line too.

Reply:
The latter both are done in commit https://github.com/RasaHQ/rasa/pull/8926/commits/dffbbe2f725cd90fe9de9fb26388f4852def26f4 .

Changelog added in https://github.com/RasaHQ/rasa/pull/8926/commits/d83d0462d050d192f863b0ebd8ec96d07aaf2d04 .

Reply:
But now that I enforced black upon the domain.py it also changed a lot of line breaks of code of other contributors. That ok for you?

Reply:
That's fine - please check the CI Code Quality fail messages, these will need to be fixed.

Reply:
flake8 instead of black, see below.